Journalator/Source/Monitor/InvestONator.lua

```
@@ -0,0 +1,42 @@
+  self:RegisterEvent("AUCTION_HOUSE_BROWSE_RESULT_UPDATED")
+  self:RegisterEvent("AUCTION_HOUSE_CLOSED")
+  self:RegisterEvent("ITEM_PURCHASED")
+end
+
+function JournalatorMonitorInvestONatorMixin:OnEvent(eventName, ...)
+  if eventName == "ITEM_PURCHASED" then
+    self:OnItemPurchased(...)
+  end
+end
+
+function JournalatorMonitorInvestONatorMixin:OnItemPurchased(itemID, itemLink, quantity, price)
```

Luacheck (1.2.0)

[warning] mutating non-standard global variable 'JournalatorMonitorInvestONatorMixin'

(W112)